quix-frontend/client/src/lib/runner/directives/results/console/console-result.html

```diff
@@ -1,6 +1,22 @@
-<div class="bi-c">
-  <pre ng-class="::{
-    'bi-white': row.level === 'INFO',
-    'bi-danger': row.level === 'ERROR'
-  }" ng-repeat="row in query.getResults().buffer"><span class="bi-white">[{{::row.timestamp}}]</span> {{::row.line}}</pre>
-</div>
\ No newline at end of file
+<div class="bi-c bi-theme--darker" data-hook="console-result"> 
+  <div 
+    ng-if="!$last && isTimestampVisible(row.timestamp, $index, $first, $last)"
+    ng-repeat-start="row in query.getResults().buffer"
+    class="cr-timestamp bi-s-v--x2"
```

Comment:
there seems to be some inconsistency in the class prefix, `cr` vs `console-result`